src/AzureClient/EntryPoint/EntryPointGenerator.cs

```diff
@@ -154,10 +154,32 @@ internal class EntryPointGenerator : IEntryPointGenerator
             if (EntryPointAssemblyInfo == null || logger.HasErrors)
             {
                 Logger?.LogError($"Error compiling entry point for operation {operationName}.");
-                throw new CompilationErrorsException(logger.Errors.ToArray());
+                throw new CompilationErrorsException(logger);
             }
 
-            var entryPointOperationInfo = EntryPointAssemblyInfo.Operations.Single();
+            if (EntryPointAssemblyInfo.Operations.Count() <= 1)
+            {
+                Logger?.LogWarning("Entry point assembly contained zero or one operations; this may indicate that C# code is not being correctly regenerated. At least two operations (the entry point and the operation called from the entry point) are expected.");
```

Comment:
I think this message is confusing and doesn't really tell the user what to do to fix the problem.

Reply:
Fair enough! In this particular case, the message is only written to logs and is not user-visible from within notebooks or host programs, such that my intent was for the message to be actionable by developers or contributors rather than by users. That said, can look at rewording some here.

Reply:
Fixed to clearly indicate that the log message is an indicator of an internal bug and not likely to be user-actionable beyond filing an issue.